pkg/pgmodel/metrics/database/metrics.go

```diff
@@ -0,0 +1,154 @@
+package database
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const namespace = "promscale_sql"
+
+var dbHealthErrors = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Namespace: namespace,
+		Subsystem: "database",
+		Name:      "health_check_errors_total",
+		Help:      "Total number of database health check errors.",
+	},
+)
+
+func init() {
+	prometheus.MustRegister(dbHealthErrors)
+}
+
+type metricQueryWrap struct {
+	metric        prometheus.Collector
+	query         string
+	isHealthCheck bool
+}
+
+var metrics = []metricQueryWrap{
+	{
+		metric: prometheus.NewCounter(
+			prometheus.CounterOpts{
+				Namespace: namespace,
+				Subsystem: "database",
```

Comment:
```suggestion
				Namespace: util.PromNamespace,
				Subsystem: "sql_database",
```
Let's reuse the main namespace in all metrics.